index.js

```diff
@@ -33,6 +33,7 @@ function Deps (opts) {
         });
     };
     this.cache = opts.cache;
+    this.acorn = {parse: {acorn: opts.acorn}};
```

Comment:
this `acorn` the best name for this option? and why is it nested in `parse`? i can't imagine a "right answer", just wondering your thoughts.